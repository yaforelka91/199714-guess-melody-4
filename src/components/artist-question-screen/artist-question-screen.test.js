import React from 'react';
import renderer from 'react-test-renderer';
import ArtistQuestionScreen from './artist-question-screen.jsx';

const question = {
  type: `artist`,
  song: {
    artist: `Singer 1`,
    src: `https://upload.wikimedia.org/wikipedia/commons/4/4e/BWV_543-fugue.ogg`,
  },
  answers: [
    {
      picture: `http://placekitten.com/134/134`,
      artist: `Singer 1`,
    }, {
      picture: `http://placekitten.com/134/134`,
      artist: `Singer 2`,
    }, {
      picture: `http://placekitten.com/134/134`,
      artist: `Singer 3`,
    }
  ],
};

describe(`ArtistQuestionScreenSnapshots`, () => {
  it(`should render ArtistQuestionScreen`, () => {
    const tree = renderer.create(
        <ArtistQuestionScreen
          question={question}
          onAnswer={() => {}}
          renderPlayer={() => {}}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
