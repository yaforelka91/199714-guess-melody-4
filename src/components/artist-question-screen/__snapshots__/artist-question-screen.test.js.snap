// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArtistQuestionScreenSnapshots should render ArtistQuestionScreen 1`] = `
<section
  className="game__screen"
>
  <h2
    className="game__title"
  >
    Кто исполняет эту песню?
  </h2>
  <div
    className="game__track"
  >
    <div
      className="track"
    />
  </div>
  <form
    className="game__artist"
  >
    <div
      className="artist"
    >
      <input
        className="artist__input visually-hidden"
        id="answer-0"
        name="answer"
        onChange={[Function]}
        type="radio"
        value="artist-0"
      />
      <label
        className="artist__name"
        htmlFor="answer-0"
      >
        <img
          alt="Singer 1"
          className="artist__picture"
          src="http://placekitten.com/134/134"
        />
        Singer 1
      </label>
    </div>
    <div
      className="artist"
    >
      <input
        className="artist__input visually-hidden"
        id="answer-1"
        name="answer"
        onChange={[Function]}
        type="radio"
        value="artist-1"
      />
      <label
        className="artist__name"
        htmlFor="answer-1"
      >
        <img
          alt="Singer 2"
          className="artist__picture"
          src="http://placekitten.com/134/134"
        />
        Singer 2
      </label>
    </div>
    <div
      className="artist"
    >
      <input
        className="artist__input visually-hidden"
        id="answer-2"
        name="answer"
        onChange={[Function]}
        type="radio"
        value="artist-2"
      />
      <label
        className="artist__name"
        htmlFor="answer-2"
      >
        <img
          alt="Singer 3"
          className="artist__picture"
          src="http://placekitten.com/134/134"
        />
        Singer 3
      </label>
    </div>
  </form>
</section>
`;
